Apps/Sandcastle/gallery/Physically-Based Materials.html

```diff
@@ -53,7 +53,7 @@
 viewer.scene.globe.depthTestAgainstTerrain = true;
 
 var position = new Cesium.Cartesian3(-1371108.6511167218, -5508684.080096612, 2901825.449865087);
-var heading = Cesium.Math.toRadians(90);
+var heading = Cesium.Math.toRadians(180);
```

Comment:
Point of trivia, this change actually puts the heading back to [what is was prior to #6592](https://github.com/AnalyticalGraphicsInc/cesium/pull/6592/files#diff-4ff54e0300165793e287a71858f79a7cL54), which is not a coincidence.  The model was updated to the correct glTF 2.0 orientation in that PR, but Cesium wasn't yet handling the 2.0 orientations correctly, so this was a workaround that is now being removed.